### PR TITLE
Allow symfony 5 components

### PR DESCRIPTION
**Pull request by ahoendgen**
Allow symfony 5 console component

### CODE REVIEW

File: src/Commands/HookCommand.php [hunk 1]
```diff
@@ -23,13 +23,19 @@ protected function configure()
         $this
             ->setName($this->hook)
             ->setDescription("Test your {$this->hook} hook")
-            ->setHelp("This command allows you to test your {$this->hook} hook")
-        ;
+            ->setHelp("This command allows you to test your {$this->hook} hook");
```
Comment (BrainMaestro) [inline src/Commands/HookCommand.php]: can you undo this? the `;` was intentionally placed like that

File: src/Commands/HookCommand.php [hunk 2]
```diff
@@ -23,13 +23,19 @@ protected function configure()
...
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $contents = is_array($this->contents) ? implode(PHP_EOL, $this->contents) : $this->contents;
-        $output->writeln(shell_exec($contents));
+
+        $outputMessage = [];
+        $returnCode    = 0;
+        exec($contents, $outputMessage, $returnCode);
+
+        $output->writeln(implode("\n", $outputMessage));
```
Comment (BrainMaestro) [inline src/Commands/HookCommand.php]: can you use `PHP_EOL` instead of `"\n"`

File: src/Commands/HookCommand.php (hunk 1 above)
Comment (ahoendgen) [inline src/Commands/HookCommand.php]:
```suggestion
            ->setHelp("This command allows you to test your {$this->hook} hook")
            ;
```

File: src/Commands/HookCommand.php (hunk 2 above)
Comment (ahoendgen) [inline src/Commands/HookCommand.php]:
```suggestion
        $output->writeln(implode(PHP_EOL, $outputMessage));
```

**Comment by shiftby** [comment]
Hi @BrainMaestro, do you possibly have a release date for this feature? Thanks!

**Comment by BrainMaestro** [comment]
Hi @shiftby I will do a release in the next few days

**Comment by BrainMaestro** [comment]
@shiftby done. https://github.com/BrainMaestro/composer-git-hooks/releases/tag/v2.8.3

**Comment by shiftby** [comment]
Thanks

**Pull request merged**